clang/lib/AST/CommentParser.cpp

```diff
@@ -295,6 +367,7 @@ Parser::parseCommandArgs(TextTokenRetokenizer &Retokenizer, unsigned NumArgs) {
       Comment::Argument[NumArgs];
   unsigned ParsedArgs = 0;
   Token Arg;
+
```

Comment:
Can you revert whitespace only changes?

Reply:
Fixed, I try to keep the clang-format changes restricted to only the regions I touch but it looks like some sneaked through